src/export/exporter.ts

```diff
@@ -86,4 +120,18 @@ export abstract class AzureMonitorBaseExporter implements BaseExporter {
 
     return filteredEnvelopes;
   }
+
+  private _sendFirstPersistedFile() {
+    this._persister.shift((err, envelopes) => {
+      if (err) {
+        this._logger.warn(`Failed to fetch persisted file`, err);
+      } else if (envelopes) {
+        this._sender.send(envelopes, () => {
+          /** no-op */
+        });
+      } else {
+        this._logger.info(`No file was found in persistent storage`);
```

Comment:
Do you want to log this? could pollute the logger I guess

Reply:
removing